content/docs/dvclive/get-started.md

```diff
@@ -66,40 +93,15 @@ dvclive        train.py
 dvclive.json
```

Comment:
I think it's hard to visualize the directory structure of the outputs for the various log methods. One way to clarify is to show the full directory structure here if all different log methods are used:

```
$ tree
├── dvclive
│   ├── images
│   │   └── img.png
│   ├── plots
│   │   └── roc.json
│   └── scalars
│       └── acc.tsv
└── dvclive.json
```

This might be too much or confusing, though, in which case I think we can show examples under each log method.

Reply:
> This might be too much or confusing, though, in which case I think we can show examples under each log method.

I don't feel like it's too much for the get started. I think I will add tree examples in both places